src/index.html

```diff
@@ -1,13 +1,37 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <title>Product cards</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <section class="products">
+    <div class="products__item cart" data-qa="card">
+      <img src="images/imac.jpeg" alt="IMac" class="cart__photo" width="160" height="134">
+      <h3 class="cart__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+      <p class="cart__code">Код товара: 195434</p>
+      <div class="user-voice">
+        <div class="cart__rating">
+          <span class="cart__ratings cart__rating--active"></span>
+          <span class="cart__ratings cart__rating--active"></span>
+          <span class="cart__ratings cart__rating--active"></span>
+          <span class="cart__ratings cart__rating--active"></span>
+          <span class="cart__ratings"></span>
+        </div>
```

Comment:
You dont need any special classes, just stars--4 or etc in parrent container as it was in "stars" task